### ISSUE TITLE
Tooltip Field Order

### ISSUE THREAD

**Issue by ramoher1** [bhavesh-jadav/Advance-Card#35]
**Describe the bug**
When using the Tooltip fields, the order of fields is not displayed in the same order as they are in the Fields section. 

**To Reproduce**
1. Add new Advance Card visualization
2. Add whole number field to "Data field" (ex. Unique Visitors)
3. Add calculated metric to "Tooltip field" (ex. Unique Visitors YOY%)
4. Add field in "Data field" to "Tooltip field" (ex. Unique Visitors) below current tooltips
5. Add calculated metric to "Tooltip field" (ex. Unique Visitors Last Year) below current tooltips
6. Click into Format section
7. Click "Show Tooltip" slider to "On"
8. Hover over visualization to see display order of fields

**Expected behavior**
When card is hovered over, tooltips should be displayed in the order shown in the Fields section

**Screenshots**
Expected Order
![image](https://user-images.githubusercontent.com/48135063/53659167-728d6580-3c20-11e9-9a03-007f5281167c.png)

Displayed Order
![image](https://user-images.githubusercontent.com/48135063/53659247-acf70280-3c20-11e9-9357-e30088eb9439.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Power BI Desktop
 - Version: 2.66.5376.2521 64-bit (February 2019)

**Comment by bhavesh-jadav** [comment]
Did you trying droping fields in tooltip in different order?

**Comment by ramoher1** [comment]
Yes, I tried reordering them initially. After that didn’t work, I removed all of the fields in the tooltips and dropped them in in the order I wanted, but the same error occurred.

**Comment by bhavesh-jadav** [comment]
I tried same thing. I dropped the field in correct order and it diplayed in correct order.
![](https://i.imgur.com/BoemxPo.gif)
But if you dropped them in wrong order and tried to rearrange them, it will show them in intial order in which you dropped them.
![](https://i.imgur.com/wIEe3v9.gif)

I suggest you try this on new card and see if it shows the same results and let me know.

**Comment by ramoher1** [comment]
That worked, thank you. I wish it would update just by moving them around rather than making a whole new card.
